openedx/adg/common/lib/mandrill_client/client.py

```diff
@@ -20,11 +22,13 @@ class MandrillClient(object):
     USER_ACCOUNT_ACTIVATION = 'adg-user-activation-email'
     VERIFY_CHANGE_USER_EMAIL = 'adg-verify-email-address-change-step-2'
     APPLICATION_SUBMISSION_CONFIRMATION = 'adg-application-submission-confirmation-1'
+    APPLICATION_WAIT_LISTED = 'adg-waitlisted-application'
```

Comment:
I kept them separate because it is being show in the pycharm is typo, but because it is being used the same way at many places, so I agree we should keep it consistent

Reply:
I kept them separate because it is being show in the pycharm is typo, but because it is being used the same way at many places, so I agree we should keep it consistent